packages/flutter_tools/lib/src/base/os.dart

```diff
@@ -607,7 +607,17 @@ enum HostPlatform {
   darwin_arm64,
   linux_x64,
   linux_arm64,
-  windows_x64,
+  windows_x64;
+
+  String get platformName {
+    return switch (this) {
+      HostPlatform.darwin_x64 => 'x64',
```

Comment:
Should these strings be constants that are used in multiple places in this pr?

Reply:
I don't think we gain a lot there. Especially w/ the CLI tool. It'd just add complexity, me thinks.